serverlessworkflow/modules/ROOT/pages/getting-started/getting-started-modules/assembly-create-your-first-workflow-service.adoc

```diff
@@ -0,0 +1,52 @@
+:_content-type: ASSEMBLY
+:description: Creating first workflow service
+:keywords: creating, serverless, workflow
+:navtitle: Creating your first workflow service
+:page-aliases:
+
+[id="assembly-create-your-first-workflow-service_{context}"]
+= Creating your first workflow service
+
+As a developer, you can use {product_name} and create a `Hello World` application, which includes the following procedures:
+
+* <<proc-boostrapping-a-project,Bootstrapping a project>>
+* <<proc-creating-a-workflow,Creating a workflow>>
+* <<proc-building-your-workflow-application,Building your workflow application>>
+* <<proc-running-your-workflow-application,Running your workflow application>>
+* <<proc-testing-your-workflow-application,Testing your workflow application>>
+
+Also, you can directly access an example application by cloning `serverless-workflow-examples/serverless-workflow-hello-world` from the link:{kogito_sw_examples_url}/serverless-workflow-hello-world[GitHub repository].
+
+This document describes how to create a workflow application that serves a `hello_world` endpoint. The workflow contains the following two states:
+
+* `Inject Hello World`: Injects a `Hello World` message into the response
+* `Inject Mantra`: Injects a `Mantra` message into the response
+
+.Example Hello World workflow
+image::getting-started/hello-world-workflow.png[]
```

Comment:
image does not show
![image](https://github.com/kiegroup/kogito-docs/assets/17157711/29c8991c-f1c1-4b9a-b33b-d9098ebae7e6)